controllers/profileController.js

```diff
@@ -0,0 +1,109 @@
+const appError = require('../utils/appError')
+const logger = require('../utils/logger')
+const catchAsync = require('../utils/catchAsync')
+const cloudinary = require('cloudinary').v2
+const fs = require('fs')
+const db = require('../models')
+const Profile = db.profile
+require('dotenv').config()
+
+cloudinary.config({
+    cloud_name: process.env.CLOUDINARY_CLOUD_NAME,
+    api_key: process.env.CLOUDINARY_API_KEY,
+    api_secret: process.env.CLOUDINARY_SECRET
+})
+
+
+const createProfile = catchAsync(async (req,res) => {
+    // get req body and files
```

Comment:
You and this you catchAsync you sha no wan use ‘express-async-error’ sha come teach me

Reply:
lmao. i no sabi this express-async-error. I go start use am, no worry,

Reply:
I got rid of catchAsync and used 'express-async-error'. Check it out